helm/etcd-operator/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: etcd-operator
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
```

Comment:
I guess this should be updated together for the every release

Reply:
Sure, we will think about that.

Reply:
It's updated on every release, here's a [pipeline](https://github.com/aenix-io/etcd-operator/blob/44f0165548004347ea48e58ae043a369dc9d073b/.github/workflows/helm-publish.yaml).

Reply:
Got it, thank you.

Another question to note: should it be `v0.0.1` or `0.0.1`?

Reply:
It should be the same as OCI-image tag, in our case `v0.0.1`.
I've set it correctly:
- In [chart](https://github.com/aenix-io/etcd-operator/blob/44f0165548004347ea48e58ae043a369dc9d073b/charts/etcd-operator/Chart.yaml#L5)
- In [pipeline](https://github.com/aenix-io/etcd-operator/blob/44f0165548004347ea48e58ae043a369dc9d073b/.github/workflows/helm-publish.yaml#L56)